app/forms/candidate_interface/degree_year_form.rb

```diff
@@ -37,5 +38,9 @@ def award_year_is_before_the_end_of_next_year
 
       errors.add(:award_year, :greater_than_limit, date: upper_year_limit) if award_year.to_i >= upper_year_limit
     end
+
+    def award_year_is_before_training_starts
+      errors.add(:award_year, :in_time_for_training) if degree.predicted_grade? && award_year.to_i >= RecruitmentCycle.current_year + 2
```

Comment:
````
 errors.add(:award_year, :in_time_for_training) if degree.predicted_grade? && award_year.to_i >= RecruitmentCycle.current_year + 2

````

Would this not be  degree.predicted_grade? && award_year.to_i <=  RecruitmentCycle.current_year

Rn the current_year is 2021 so they would need to graduate in 2021 in order to start a course in September